rfcs/text/0001-rfc-process.md

```diff
@@ -119,6 +193,14 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ### Reviewing RFCs
 
+**ADDITION BEGINS**
+
+As a general guidance for reviewers, we recommend (but not require) the following rules of etiquette:
+
+- For Technical RFCs, w 
```

Comment:
_w_? Incomplete rule.

Also recommend (should) means _not require_  (must) so that's redundant. 

I'm wondering whether we should include the _boilerplate_ in the RFC template:

> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14, RFC2119, and RFC8174 when, and only when, they appear in all capitals, as shown here.

and be more formal about our use of key words.

Reply:
Oops! Yes, I had thought of that. I think it's worthwhile so will make the change.